homeassistant/components/rfxtrx.py

```diff
@@ -318,16 +318,32 @@ def _send_command(self, command, brightness=0):
                 self._event.device.send_on(RFXOBJECT.transport)
             self._state = True
 
-        elif command == "dim":
+        if command == "dim":
             for _ in range(self.signal_repetitions):
                 self._event.device.send_dim(RFXOBJECT.transport,
                                             brightness)
             self._state = True
 
-        elif command == 'turn_off':
+        if command == 'turn_off':
             for _ in range(self.signal_repetitions):
                 self._event.device.send_off(RFXOBJECT.transport)
             self._state = False
             self._brightness = 0
 
+        if command in ["roll_up", "roll_down", "stop_roll"]:
+            self._send_rollershutter_command(command)
+
         self.update_ha_state()
+
+    def _send_rollershutter_command(self, command):
```

Comment:
I agree, however the linter wasn't happy. `too-many-branches` error. Needed to separate into multiple functions.